src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -136,45 +190,50 @@ export class AttachedAccountsTreeItem implements IAzureParentTreeItem {
                 const accounts: any[] = JSON.parse(value);
                 await Promise.all(accounts.map(async account => {
                     let id: string;
+                    let label: string;
```

Comment:
ID would be `localhost:8081`, label would be `localhost:8081 (Emulator - MongoDB)`.